mindmap/static/js/src/mindmap.js

```diff
@@ -39,11 +86,127 @@ function MindMapXBlock(runtime, element, context) {
+              `;
+
+              $(element).find(".modal__data").html(dataTableHTML);
+              $(element).find("#modal_title").html("Mindmap submissions");
+
+              const dataTable = $("#dataTable").DataTable({
+                data: assignments,
+                scrollY: "50vh",
+                dom: "Bfrtip",
+                bPaginate: false,
+                destroy: true,
+                columns: [
+                  { data: "username" },
+                  { data: "timestamp" },
+                  { data: "submitted" },
+                  { data: "score" },
+                  {
+                    data: null,
+                    render: function () {
+                      return '<button class="review_button button-link" type="button">Review</button>';
+                    },
+                  },
+                ],
+              });
+
+              handleRowDataTableClick(dataTable);
+            }
+
+            function handleRowDataTableClick(dataTable) {
+              $(element)
+                .find("#dataTable")
+                .on("click", ".review_button", function (e) {
+                  e.preventDefault();
+                  const target = $(e.target);
+                  const link = target;
+                  const rowReview = link.closest("tr");
+                  const submissionData = dataTable.row(rowReview).data();
+                  const answerMindMap = submissionData.answer_body.mindmap_student_body;
+                  const answerMindMapFormat = JSON.parse(answerMindMap);
+
+                  const mindMapReviewContainer = `
+                    <div class="review_mindmap_container">
+                      <button class="button-link back-review">&larr;&nbsp; Back</button>
+                      <div id="review-mindmap"></div>
+                      <div class="grade-assessment">
+                        <form id="grade-assessment-form">
+                          <div class="grade-assessment_form-control">
+                            <label for="grade">Grade</label>
+                            <input type="number" name="grade" class="inputs-styles" /> 
+                          </div>
+                          <div class="grade-assessment_form-control">
+                            <label for="comment">Comment</label>
+                            <textarea rows="4" cols="50" name="comment" class="text-area-styles" placeholder="optional"></textarea>
+                          </div>
+                          <div class="grade-assessment_form-buttons">
+                            <button type="submit" class="grade-assessment__button-submit">Submit</button>
+                            <button type="submit" class="grade-assessment__button-submit">Remove grade</button>
+                          </div>
+                        </form>
+                      </div>
+                    </div>`;
+
+                  const modalTitle = `Reviewing ${submissionData.username}'s Mindmap`;
```

Comment:
We need to mark this (and the rest, like the headers) as translatable strings:
```suggestion
                  const modalTitle = gettext('Reviewing Mindmap for student:') + username;
```
The suggestion's syntax is probably wrong, but I hope you get the gist. After marking all strings to translate, follow [these instructions](https://github.com/eduNEXT/xblock-mindmap/#translations) to generate the new .mo files.

@BryanttV am I missing something or is this the correct procedure?